packages/matter-node.js/README.md

```diff
@@ -61,6 +61,17 @@ or when starting from TS files:
 ```bash
 npm run matter -- -on "echo 255 > /sys/class/leds/led1/brightness" -off "echo 0 > /sys/class/leds/led1/brightness"
 ```
+(Please note the "--" to separate commandline parameters between the npm run and the executed script. 
+
+The following parameters are available:
+* -passcode: the passcode to use for pairing (default: 20202021)
+* -discriminator: the discriminator to use for pairing (default: 3840)
+* -vendorid: the vendor ID as number to use for pairing (default: 65521 (0xFFF1))
+* -productid: the product ID as number to use for pairing (default: 32768 (0x8000))
+* -port: the port to listen on (default: 5540)
+* -file: the file to use for storing the pairing information (default: device.json). Delete the file or provide an alternative name to reset the device
```

Comment:
The default was device.json before ... I adjusted it to be "device-node" and "contreoler-node".

I decided against a number in the name because formally the nodeid for "device usecase" is disctated by the controller that pairs, so so "node 1" will not be "node id 1". That might be misleading.
For Controller we could do that because there we will use ID 1 by default currently, but also will be randomized in next step, so would also just confuse.

Reply:
I'm not seeing the update to `device-node` in the code or docs.  Perhaps my pull of the latest version of this PR didn't take.  Either way, this can be a follow-on if it needs correcting.